src/dso_api/dynamic_api/serializers.py

```diff
@@ -799,7 +799,20 @@ def _through_serializer_factory(m2m_field: models.ManyToManyField) -> type[Throu
     through_model = m2m_field.remote_field.through
     target_model = m2m_field.related_model
     target_table_schema = m2m_field.related_model.table_schema()
-    target_fk_name = m2m_field.m2m_reverse_field_name()  # second foreign key of the through model
+    try:
+        target_fk_name = (
+            m2m_field.m2m_reverse_field_name()
+        )  # second foreign key of the through model
```

Comment:
```suggestion
        # second foreign key of the through model
        target_fk_name = m2m_field.m2m_reverse_field_name()
```
better have the comment above due to the black formatting change